WooCommerce/Classes/ViewRelated/Coupons/CouponManagementViewController.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+class CouponManagementViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigation()
+    }
+}
+
+// MARK: - View Configuration
+//
+private extension CouponManagementViewController {
+
+    func configureNavigation() {
+        title = NSLocalizedString("Coupons", comment: "Coupon management coupon list screen title")
```

Comment:
What do you think about moving this localized string under an enum called `Localization`? We use it in many points of the app.